src/session/content/message_row/sticker.rs

```diff
@@ -48,48 +44,48 @@ mod imp {
 
 glib::wrapper! {
     pub struct MessageSticker(ObjectSubclass<imp::MessageSticker>)
-        @extends gtk::Widget;
-}
-
-impl Default for MessageSticker {
-    fn default() -> Self {
-        Self::new()
-    }
+        @extends gtk::Widget, MessageRow;
 }
 
 impl MessageSticker {
-    pub fn new() -> Self {
-        glib::Object::new(&[]).expect("Failed to create MessageSticker")
+    pub fn new(message: &glib::Object) -> Self {
+        glib::Object::new(&[("message", message)]).expect("Failed to create MessageSticker")
     }
```

Comment:
Maybe this function is a little bit redundant across all subclasses of `MessageRow`.
It seems to me that all subclasses are going to be initialized with a message object. So it makes sense,
to put this in `MessageRowExt` trait.

I have done this roughly in the attached patch. As a positive side effect, the method `ItemRow::create_message_row` can then look much cleaner.

[fn_new_in_base.zip](https://github.com/melix99/telegrand/files/7715645/fn_new_in_base.zip)

Reply:
Great, thanks! :)